features/logging/cluster_log_forwarder.feature

```diff
@@ -139,6 +139,7 @@ Feature: cluster log forwarder features
   @destructive
   @singlenode
   @4.7 @4.6
+  @logging5.6
```

Comment:
Are we only planning to run 5.6 on prow?

Reply:
After the necessary steps for logging on Prow are added in the PR, which currently contains 5.6, we will simply need to add the Logging 5.7 configuration. In the meantime, I've went ahead and added the logging5.7 tag.

Reply:
Not sure if we are planning to support 5.5.z runs on prow currently. My guess would be yes but we could update it later with a separate PR once we test 5.6 on prow.